<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewTeam</class>
 <widget class="QDialog" name="addNewTeam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.error1,.error2,.error3,.error4,.error5
{
 color:#ff0000;
}</string>
  </property>
  <widget class="QLineEdit" name="teamName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Team Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stadium Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stadiumName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>City, State, and Zip Code</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cityStateZip">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="seatingCapacity">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seating Capacity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>National League</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>American League</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nationalLeague">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>391</y>
     <width>140</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="americanLeague">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>140</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Street Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="streetAddress">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneNumber">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Date Opened (Month, day and year)</string>
   </property>
  </widget>
  <widget class="QLabel" name="error1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>required *</string>
   </property>
  </widget>
  <widget class="QLabel" name="error2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>required *</string>
   </property>
  </widget>
  <widget class="QLabel" name="error3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>required *</string>
   </property>
  </widget>
  <widget class="QLabel" name="error4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>required *</string>
   </property>
  </widget>
  <widget class="QLabel" name="error5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>required *</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthDayYear">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
